docs/introduction/installation.mdx

```diff
@@ -7,130 +7,115 @@ description: Pester is a cross-platform PowerShell module for testing your Power
-Unfortunately there are some complications specific to Pester that we cannot avoid.
+### Linux & MacOS
 
-Pester version 3.4.0 ships as part of Windows 10 and Windows server 2016, and that distribution conflicts with the standard module update mechanism.
-It is not possible to update the built-in Pester to a newer version, using the `Update-Module` cmdlet.
-
-Instead you need to perform a new side-by-side installation of Pester, because `Install-Module` detects the current installation of Pester is signed with a different signature than the one you are installing.
-
-Here's the command you need to run _as administrator_ in order to get the latest version of Pester:
+From a PowerShell session run:
 
 ```powershell
-Install-Module -Name Pester -Force -SkipPublisherCheck
-```
-
-The `-SkipPublisherCheck` option is typically required for machines that only have v3.4.0 installed as mentioned above.
-That particular version of the module was signed by Microsoft to be shipped in-box, which is not the case for later versions which are community maintained and signed with a different certificate.
-Attempting to install the module without that option may fail.
-
-If you receive the following warning when trying to install the module, you may need to explicitly enable TLS 1.2:
+# For all users
+Install-Module -Name Pester
 
-```powershell
-WARNING: Source location 'https://www.powershellgallery.com/api/v2/packages/Pester/4.10.1' is not valid
+# For current user only
+Install-Module -Name Pester -Scope CurrentUser
 ```
 
-To enable TLS 1.2 for the current PowerShell instance, you can run the following code, and then re-run the installation command:
+To update:
 
 ```powershell
-[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor [System.Net.SecurityProtocolType]::Tls12
+Update-Module -Name Pester
 ```
 
-### Removing the Built-In Version of Pester
+### Windows
+
+Windows 10 / Windows Server 2016 and later ships with Pester version 3.4.0. This built-in version cannot be updated using the simple `Update-Module` cmdlet.
+Instead you need to perform a side-by-side installation to get started with the latest version of Pester.
 
-If you need to remove the built-in version of Pester on your machine, you can do so by executing the following code:
+The built-in version is signed by Microsoft while newer versions are community-maintained and signed with a different certificate,
+causing `Install-Module` to sometimes throw a error requiring us to accept the new publisher certificate.
 
+Run the command below to install the latest version:
 ```powershell
-$module = "C:\Program Files\WindowsPowerShell\Modules\Pester"
-takeown /F $module /A /R
-icacls $module /reset
-icacls $module /grant "*S-1-5-32-544:F" /inheritance:d /T
-Remove-Item -Path $module -Recurse -Force -Confirm:$false
+# -Force to install side-by-side
+# -SkipPublisherCheck to accept the newer certificate
```

Comment:
Publisher check error was PowerShellGet 1.6.5 shipped in PowerShell 6.1 and later.
Still including the `-SkipPublisherCheck` switch as all versions of Windows ship with an older PowerShellGet 1.0.0.1 version for Windows PowerShell.